### DISCUSSION TITLE
Support dot (.) in alias prefix

### DISCUSSION THREAD

**Discussion by nguyenkims** [simple-login/app#414] [Feature Requests]
Moved from https://trello.com/c/4EtvhofF/123-support-dot-in-alias-prefix

**Comment by ITechGeek81** [comment]
I don't understand why this isn't available w/ manually created aliases.  My auto-created aliases made via the catch-all addresses have no issues w/ periods.  The issue w/ the period seems to strictly be in the front-end interface.

**Reply by nguyenkims** [reply]
Hi the main reason is that some special characters like dot sign (.) are reserved for future uses. Currently we don't really have a use case that would need to dot sign though so I think supporting dot sign in alias prefix should be straightforward.

**Comment by bingoxo** [comment]
I have encountered many websites rejecting the alias when (+) is used , so please implement (.) asap :)

**Reply by pocki** [reply]
I also have encountered many websites where only dot (.), underline (_) or minus (-) where possible.

**Comment by ITechGeek81** [comment]
Period (.) is a very common character in email addresses, especially in a case where someone is using their name.

In my case, I like to create email addresses as <domain.tld>@<my domain>, I only found out the period wasn't available because I wanted to pre-create a email address for a service to set update it's destinations before hand.

**Reply by codywd** [reply]
Yeah, I have several domains where I use first.last@domain.com. I switched them from various providers (MXLogin primarily) to SimpleLogin. Just easier to manage a single inbox. But I can no longer do first.last, so it's firstlast right now

**Reply by codywd** [reply]
Funny enough, if you turn on the catch-all feature for your custom domain (not sure if it works on non-custom domains), it'll create an email with a dot in it just fine.

**Comment by nguyenkims** [comment]
Just to let you know that the dot sign is now supported in the latest SaaS version (aka simplelogin.io) and will be included in the next  self-hosted version :)

![2021-04-30 at 17 03](https://user-images.githubusercontent.com/3167780/116715873-673c4d00-a9d7-11eb-84e4-d96b044f3f1d.png)

**Reply by codywd** [reply]
Awesome! Thank you! :)

**Reply by bingoxo** [reply]
Is it supported with directories ?

**Reply by nguyenkims** [reply]
Yes you can create aliases like `{directory}/ab.cd@simplelogin.co`

**Reply by bingoxo** [reply]
Not really useful to be honest , somrthing like directory.abcd@simplelogin.co would be more useful , since directory/abcd is already supported

**Reply by nguyenkims** [reply]
Oh directory already supports the plus sign (+) as separator as you can see on https://simplelogin.io/blog/alias-directory/

**Reply by bingoxo** [reply]
sorry i meant directory.abcd , as i said before many sites dont like the + sign

**Reply by nguyenkims** [reply]
Oh I see, the dot sign in this case can't be distinguished with the current custom alias creation unfortunately.

**Reply by bingoxo** [reply]
i think the use case i mentioned is more important than what's implemented now or maybe you can make a poll about it

**Reply by jdahdah** [reply]
Thank you! Would you please also enable this in the extension?

![CleanShot 2022-04-15 at 13 32 55@2x](https://user-images.githubusercontent.com/5282190/163566027-ab546fb5-35bb-4ed1-b23b-a62b732957f9.png)

**Reply by nguyenkims** [reply]
@ntnhon do you think we can release a new version of the safari extension that supports dot sign (.) in alias prefix? We haven't started converting the chrome/firefox extension to safari one yet so in the meantime, we still need to support the safari one ...

**Reply by nguyenkims** [reply]
The new safari extension now supports dot sign (.) in alias prefix
The new extension can be downloaded on https://apps.apple.com/app/id6475835429 

![CleanShot 2024-03-14 at 11 13 17](https://github.com/simple-login/app/assets/3167780/9dff0db6-3227-4c39-83f4-4261a55b0302)
